app/views/permissions/_hidden_collections_notification.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless @hidden_entry_titles.blank? %>
```

Comment:
Agreed. I had already added this when you said that @mreese84 wouldn't want a message to the user, so I left it in so we could discuss.